USE_CASES.md

```diff
@@ -171,6 +172,186 @@ print(response.body)
 print(response.headers)
 ```
 
+<a name="hello_email_django_on_heroku"></a>
+# Deploy a Simple Hello Email Django App on Heroku
+
+This tutorial explains how we set up a simple Django app to send an email with the SendGrid Python SDK and how we deploy our app to Heroku.
+
+## Create a Django project
+
+We first create a project folder.
+
+```
+$ mkdir hello-sendgrid
+$ cd hello-sendgrid
+```
+
+We assume you have created and activated a [virtual environment](https://virtualenv.pypa.io/) (See [venv](https://docs.python.org/3/tutorial/venv.html) for Python 3+) for isolated Python environments.
+
+Run the command below to install Django, Gunicorn (a Python WSGI HTTP server), and SendGrid Python SDK.
+
+```
+$ pip install django gunicorn sendgrid
+```
+
+It is a good practice for Python dependency management. We will pin the requirements with a file `requirements.txt`.
+
+```
+$ pip freeze > requirements.text
+```
+
+Run the command below to initialize a Django project.
+
+```
+$ django-admin startproject hello_sendgrid
+```
+
+The folder structure should look like this:
+
+```
+hello-sendgrid
+├── hello_sendgrid
+│   ├── hello_sendgrid
+│   │   ├── __init__.py
+│   │   ├── settings.py
+│   │   ├── urls.py
+│   │   └── wsgi.py
+│   └── manage.py
+└── requirements.txt
+```
+
+Let's create a page to generate and send an email to a user when you hit the page.
+
+We first create a file `views.py` and put it under the folder `hello_sendgrid/hello_sendgrid`. Add the minimum needed code below.
+
+```python
+import os
+
+from django.http import HttpResponse
+
+import sendgrid
+from sendgrid.helpers.mail import *
+
+
+def index(request):
+    sg = sendgrid.SendGridAPIClient(
+        apikey=os.environ.get('SENDGRID_API_KEY')
+    )
+    from_email = Email('test@example.com')
+    to_email = Email('test@example.com')
```

Comment:
Should we suggest they put their own email in the to_email variable, so that they can see the email they receive?

Reply:
I've added a note below this code already.